src/components/Input.tsx

```diff
@@ -71,36 +84,56 @@ const Input: FC<Props> = ({
     if (e.key === 'Enter' && !e.shiftKey) {
       // do not propagate keypress event when only enter is pressed
       e.preventDefault();
-      // send message
-      onClick();
+      if (!isMessageTooLong) {
+        // send message
+        onClick();
+      }
     }
   };
 
   return (
-    <Box
-      display="flex"
-      className={classes.wrapper}
-      justifyContent="center"
-      alignItems="center"
-      id={id}
-    >
-      <TextField
-        data-cy={inputTextFieldCypress}
-        id={inputTextFieldTextAreaCypress}
-        inputRef={inputRef}
-        value={textInput}
-        onChange={onChange}
-        onKeyDown={keyDown}
-        variant="outlined"
-        fullWidth
-        multiline
-        maxRows={MAX_ROWS_INPUT}
-        placeholder={placeholder || t(CHATBOX.INPUT_FIELD_PLACEHOLDER)}
-      />
-      <IconButton data-cy={sendButtonCypress} onClick={onClick}>
-        <SendIcon color="primary" />
-      </IconButton>
-    </Box>
+    <div>
+      <Box
+        display="flex"
+        className={classes.wrapper}
+        justifyContent="center"
+        alignItems="flex-end"
+        id={id}
+      >
+        <TextField
+          data-cy={inputTextFieldCypress}
+          id={inputTextFieldTextAreaCypress}
+          inputRef={inputRef}
+          value={textInput}
+          onChange={onChange}
+          onKeyDown={keyDown}
+          variant="outlined"
+          fullWidth
+          multiline
+          maxRows={MAX_ROWS_INPUT}
+          placeholder={placeholder || t(CHATBOX.INPUT_FIELD_PLACEHOLDER)}
+        />
+        <IconButton
+          data-cy={sendButtonCypress}
+          onClick={onClick}
+          disabled={isMessageTooLong}
+        >
+          <SendIcon color={isMessageTooLong ? 'disabled' : 'primary'} />
+        </IconButton>
+      </Box>
+      <Typography
+        className={clsx(classes.textLength, {
+          [classes.textTooLong]: isMessageTooLong,
+        })}
+        variant="caption"
+        data-cy={charCounterCypress}
+      >
+        {textInput
+          ? textInput.length +
+            (isMessageTooLong ? ` (max. ${MAX_MESSAGE_LENGTH_HARD} chars)` : '')
+          : ' '}
+      </Typography>
```

Comment:
I think you can factor out as `renderHelperMessage()` -> so you can beautifully return null instead of an empty string.

Reply:
The helper tex need to be a string with a single space for the node to maintain its height. Will add a comment and factor the logic out.